lib/jsdom/living/nodes/SVGUseElement.webidl

```diff
@@ -0,0 +1,12 @@
+// https://svgwg.org/svg2-draft/struct.html#InterfaceSVGUseElement
+[Exposed=Window]
+interface SVGUseElement : SVGGraphicsElement {
+  // [SameObject] readonly attribute SVGAnimatedLength x;
```

Comment:
Any reason to omit these? At least some of them seem pretty easy.

Reply:
See above:

> - Implement SVGAnimatedLength + reflection (complicated, as it requires us to implement a CSS dimension parser and unit converter – with support for relative units like em, ideally)